apypie/foreman.py

```diff
@@ -0,0 +1,228 @@
+"""
+Apypie Foreman module
+
+opinionated helpers to use Apypie with Foreman
+"""
+import time
+
+try:
+    from typing import cast, Optional, Set, Tuple
+except ImportError:
```

Comment:
Can we actually catch this? We use these methods (like `cast()`) unconditionally so I'd drop the try/except.

Reply:
This will never be thrown on Python 3, but the vendoring in FAM breaks if typing stuff is not try'ed, I'll fix that soon, but didn't yet

Reply:
https://github.com/Apipie/apypie/pull/182 is where this will be tackled